internal/network/interface.go

```diff
@@ -55,7 +55,7 @@ func (sed ServerEndpointDetails) URLForAlias(a string) string {
 	if sed.UseSubdomainForAliases {
 		u.Host = a + "." + sed.Domain
 	} else {
-		u.Path = "/alias/" + a
+		u.Path = sed.Domain + "/alias/" + a
```

Comment:
```suggestion
		u.Host = sed.Domain
 		u.Path = "/alias/" + a
```

I think it would be safer to do this instead.

Reply:
thanks for the review. I see and understand your comments.
I will be able to work on a test probably by tomorrow.